mrp_lot_production_date/models/mrp_production.py

```diff
@@ -10,6 +10,6 @@ class MrpProduction(models.Model):
     def _post_inventory(self, cancel_backorder=False):
         res = super()._post_inventory(cancel_backorder=cancel_backorder)
         for order in self:
-            if order.lot_producing_id:
+            if order.lot_producing_id and not order.lot_producing_id.production_date:
```

Comment:
Since this change was applied on the 15.0 branch it shouldn't be part of the migration commit.

Reply:
Updated :+1: